src/main/groovy/org/transmartproject/batch/concept/postgresql/PostgresInsertConceptCountsTasklet.groovy

```diff
@@ -21,19 +22,26 @@ class PostgresInsertConceptCountsTasklet extends InsertConceptCountsTasklet {
             AND sourcesystem_cd = ?
         ),
         code_patients AS (
-          SELECT concept_path, concept_cd, patient_num
+          SELECT
+            concept_path::text,
+            substring(concept_path from '#"%\\#"%\\' for '#') as parent_concept_path,
+            patient_num
           FROM
             i2b2demodata.observation_fact
             NATURAL INNER JOIN relevant_concepts
+          UNION
+          SELECT
+            parent_concept_path as concept_path,
+            substring(parent_concept_path from '#"%\\#"%\\' for '#') AS parent_concept_path,
+            patient_num
+          FROM code_patients WHERE parent_concept_path != ?
```

Comment:
Naming is confusing here. `parent_concept_path` in WHERE condition is actually `code_patients.parent_concept_path`. I am going to specify the full name (with table) instead. Please also not that there is a test case. See `testConceptCountsCorrectness `.